packages/server/src/db/tenant_creation.js

```diff
@@ -0,0 +1,235 @@
+require('dotenv').config();
```

Comment:
Nit: I suspect this might be redundant with #323

Reply:
I don't think anything bad happens if you call it multiple times, but yeah, this will be unnecessary after #323 merges.